src/AWS/Orleans.Clustering.DynamoDB/Options/DynamoDBClusteringOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Orleans.Clustering.DynamoDB;
+
+namespace Orleans.Configuration
+{
+    public class DynamoDBClusteringOptions
+    {
+        /// <summary>
+        /// AccessKey string for DynamoDB Storage
+        /// </summary>
+        public string AccessKey { get; set; }
+
+        /// <summary>
+        /// Secret key for DynamoDB storage
+        /// </summary>
+        public string SecretKey { get; set; }
```

Comment:
Add `[RedactConnectionString]`

Reply:
Here and above

Reply:
agreed, not sure how I missed those.  Waiting for tests to complete before pushing change, to see if here are any other tests I need to fix.